en/micro-integrator/docs/references/connectors/nats-connector/nats-connector-example.md

```diff
@@ -0,0 +1,198 @@
+# Core NATS
```

Comment:
Let's merge both these files into the configuration file. I think all of it belongs in the reference doc. @abeykoon WDYT?